src/lib/swapper/swappers/ThorchainSwapper/utils/getThresholdedAffiliateBps/getThresholdedAffiliateBps.test.ts

```diff
@@ -0,0 +1,116 @@
+import { Ok } from '@sniptt/monads'
+import type { AxiosResponse, AxiosStatic } from 'axios'
+import { bn } from 'lib/bignumber/bignumber'
+import { ETH } from 'lib/swapper/swappers/utils/test-data/assets'
+
+import { THORCHAIN_OUTBOUND_FEE_RUNE_THOR_UNIT } from '../../constants'
+import type { MidgardPoolResponse } from '../../types'
+import { thorService } from '../thorService'
+import {
+  getExpectedAffiliateFeeSellAssetThorUnit,
+  getOutboundFeeInSellAssetThorBaseUnit,
+  getThresholdedAffiliateBps,
+} from './getThresholdedAffiliateBps'
+
+jest.mock('../thorService', () => {
+  const axios: AxiosStatic = jest.createMockFromModule('axios')
+  axios.create = jest.fn(() => axios)
+
+  return {
+    thorService: axios.create(),
+  }
+})
+
+describe('getOutboundFeeInSellAssetThorBaseUnit', () => {
+  it('should return 0.2 rune denominated in the target asset, in thor units', () => {
+    const assetPricePrecision = '4.00000' // 1 token is 4x as valuable as 1 rune
+    expect(getOutboundFeeInSellAssetThorBaseUnit(assetPricePrecision).toNumber()).toEqual(
+      Number(THORCHAIN_OUTBOUND_FEE_RUNE_THOR_UNIT) / 4,
```

Comment:
If we want to make this explicit with comments
```suggestion
      Number(THORCHAIN_OUTBOUND_FEE_RUNE_THOR_UNIT) / 4, // .005 of the sell asset in THOR base unit
```